assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -17,12 +17,20 @@ import { withSafeTimeout } from '@wordpress/compose';
 /**
  * Internal dependencies
  */
-import { getPixelsFromPercentage } from '../../helpers';
+import {
+	getPixelsFromPercentage,
+	getPercentageFromPixels,
+	getRelativeElementPosition,
+	isCTABlock,
+} from '../../helpers';
 import {
 	STORY_PAGE_INNER_WIDTH,
 	STORY_PAGE_INNER_HEIGHT,
 } from '../../constants';
 
+const PAGE_BORDER = 50;
```

Comment:
`PAGE_BORDER` might be a bit misleading. In `EditorCarousel` there is also a `PAGE_BORDER` but it refers to the actual element's border. Here it is the space between elements. But where is that defined? Does it match some CSS somewhere? Would be good to have that mental connection documented here.

Reply:
I spent waaaaaaay too much time searching for this one only to find it was a CSS only thing. It's defined [here](https://github.com/ampproject/amp-wp/blob/develop/assets/src/stories-editor/blocks/amp-story-page/edit.css#L31) plus [here](https://github.com/ampproject/amp-wp/blob/develop/assets/src/stories-editor/blocks/amp-story-page/edit.css#L72).

And yes, it would be a good idea to document this origin. And rename it to `PAGE_SPACING` or similar.

Reply:
`STORY_PAGE_BORDER` is just as misleading as `PAGE_BORDER` IMO. "spacing" or "margin" makes more sense.

Reply:
> `STORY_PAGE_BORDER` is just as misleading as `PAGE_BORDER` IMO. "spacing" or "margin" makes more sense.

`STORY_PAGE_MARGIN` works for me.

Reply:
Fixed in b3be9f5.